packages/devtools/web/styles.css

```diff
@@ -538,6 +538,34 @@ div.tabnav {
     min-width: 500px;
 }
 
+.memory-inspector {
+    padding: 3px;
+    box-sizing: border-box;
+    font-weight: 600;
```

Comment:
why the custom font weight?

Reply:
I think that was pulled in from debugger.css.
However, 600 is odd certainly not bold (700) I'll change it to bold.

Reply:
Fixed all custom font weights 300 should be normal, 400 is normal and 600 is bold.  No more custom font-weights in our css files.